Maple2.Model/Metadata/SetItemOptionMetadata.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Maple2.Model.Enum;
```

Comment:
you should just move this whole file into: `Maple2.Model/Metadata/Table/SetItemOptionTable.cs`